libmscore/stafftype.cpp

```diff
@@ -241,6 +243,7 @@ void StaffType::write(Xml& xml) const
             xml.tag("stemsDown",        _stemsDown);
             xml.tag("stemsThrough",     _stemsThrough);
             xml.tag("upsideDown",       _upsideDown);
+            xml.tag("showTabFingering", _showTabFingering);
```

Comment:
Add an `if (_showTabFingering)` here and the default case won't get written, which should result in the mtests to pass?